connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/DedicatedMirrorIntegrationTest.java

```diff
@@ -187,6 +194,77 @@ public void testSingleNodeCluster() throws Exception {
         }
     }
 
+    @Test
+    public void testClusterWithEmitOffsetDisabled() throws Exception {
+        Properties brokerProps = new Properties();
+        EmbeddedKafkaCluster clusterA = startKafkaCluster("A", 1, brokerProps);
+        EmbeddedKafkaCluster clusterB = startKafkaCluster("B", 1, brokerProps);
+
+        try (Admin adminB = clusterB.createAdminClient()) {
+
+            // Cluster aliases
+            final String a = "A";
+            final String b = "B";
+            final String ab = a + "->" + b;
+            final String ba = b + "->" + a;
```

Comment:
Currently unused. We can either remove it, or if we want to be more explicit in our properties setup, we can use it to explicitly disable the b->a flow by setting `ba + ".enabled"` to false.